packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -251,10 +251,10 @@ export class VaRadio {
         >
           <legend part="legend">
             {label}
-            {required && <span class="required">{i18next.t('required')}</span>}
+            {required && <span class="required" part="required">{i18next.t('required')}</span>}
           </legend>
-          {hint && <span class="hint-text">{hint}</span>}
-          <span id="error-message" role="alert">
+          {hint && <span class="hint-text" part="hint">{hint}</span>}
```

Comment:
We are currently evaluating making hint text consistent across all of the form components so I think this (and the one above) should not be included at the moment: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1249

Reply:
Ok, I'll update the PR